readme.md

```diff
@@ -130,10 +130,13 @@ To use default settings globally, use `--polling={}`.
 
 You can generate the output as protocol clients, a.k.a., low level clients with `--low-level-client` flag. The models will not be generated and the methods in the clients will be generated as [protocol methods](https://github.com/Azure/azure-sdk-for-java/wiki/Protocol-Methods). `--low-leve-client` should be used in conjunction with the following settings:
 
-```yaml $(low-level-client)
+```
 generate-client-interfaces: false
 generate-client-as-impl: true
 generate-sync-async-clients: true
+add-context-parameter: true
+context-client-method-parameter: true
+sync-methods: all
 ```
```

Comment:
Remove `yaml` from the block. This part is intended to be a description of possible configures, not the actual config for autorest.